examples/kafka_lib/mux/main.go

```diff
@@ -30,7 +30,7 @@ import (
 
 func main() {
 	log.StandardLogger().SetLevel(logging.DebugLevel)
-	mx, err := mux.InitMultiplexer("", "default", logroot.StandardLogger())
+	mx, err := mux.InitMultiplexer("", "default", "", logroot.StandardLogger())
```

Comment:
please fill:
- name of the connection
- hash (from enum representing hash/manual/random)